project/Game.py

```diff
@@ -368,40 +373,40 @@ def setUp(self):
         self.game.landmarks=[l1,l2,l3]
         self.game.penaltyTime = 20
         self.game.penaltyValue = 10
-        self.game.timer = datetime.time(00,00,15)
+        self.game.timer = datetime.timedelta(hours = 0, minutes = 0, seconds = 15)
 
     def test_get_status(self):
-        self.team.clueTime = datetime.time(5,30,50)
-        currenttimecalc = (str(datetime.datetime.now().hour-self.team.clueTime.hour)+":"+str(datetime.datetime.now().minute-self.team.clueTime.minute)+":"+str(datetime.datetime.now().second-self.team.clueTime.second))
-        self.assertEqual(self.game.get_status(self.team), 'Points:100;You Are On Landmark:2;Current Time:'+currenttimecalc+';Time Taken For Landmarks:00:55:40)', 'get_status did not print the proper stats!')
+        self.team.clueTime = datetime.timedelta(hours=5,minutes=30,seconds=50)
+        currenttimecalc = (self.team.clueTime + datetime.timedelta(hours=datetime.datetime.now().hour,minutes=datetime.datetime.now().minute,seconds=datetime.datetime.now().second))
+        self.assertEqual(self.game.get_status(self.team), 'Points:100;You Are On Landmark:1;Current Time:'+str(currenttimecalc)+';Time Taken For Landmarks:0:55:40', 'get_status did not print the proper stats!')
```

Comment:
Can you explain the current time portion of the status? I’m still not getting it and why there’s so much calculation for it.